test/cypress/integration/36-pledges.text.js

```diff
@@ -0,0 +1,85 @@
+describe('Pledges', () => {
+  beforeEach(() => {
+    cy.login({ redirect: `/pledges/new` });
+  });
+
+  describe('creating an individual pledge', () => {
+    it('Create the pledge', () => {
+      cy.get('[data-cy=name]', { timeout: 600000 }).clear().type('samcaspus');
```

Comment:
Using a custom `timeout` should be an exception, not the norm. Please remove the ones that are not required.

Reply:
made the changes

Reply:
I still see a bunch of them in the code. Could you remove them or explain why they are required?

Reply:
@Betree I noticed at times the response from the api end is a bit slow which leads to failure in test. Cypress by default waits only for 4000 hence I gave it a timeout of 60000 to load the components to the Dom, so that test does not fail. Meanwhile if the response is before the allotted timeout cypress takes the element and skips the part where time out is involved.